extensions/ql-vscode/src/run-remote-query.ts

```diff
@@ -128,6 +132,29 @@ export async function runRemoteQuery(cliServer: cli.CodeQLCliServer, credentials
     void showAndLogInformationMessage(`Successfully scheduled runs. [Click here to see the progress](https://github.com/${OWNER}/${REPO}/actions).`);
 
   } catch (error) {
-    void showAndLogErrorMessage(error);
+    if (typeof error.message === 'string' && error.message.includes('Some repositories were invalid')) {
+      const invalidRepos = error?.response?.data?.invalid_repos || [];
+      const reposWithoutDbUploads = error?.response?.data?.repos_without_db_uploads || [];
+      void logger.log('Unable to run query on all repositories');
+      void logger.log('invalidRepos = ' + JSON.stringify(invalidRepos));
```

Comment:
This will look strange if `invalidRepos` is an empty array, so maybe used different text if so. Also, I generally prefer to use template strings instead of concatenation. eg-

```suggestion
      void logger.log(`invalidRepos = ${JSON.stringify(invalidRepos)}`);
```

but maybe something more text friendly would be preferable?

```suggestion
      void logger.log(`InvalidRepos: ${invalidRepos.join(', '}`);
```